src/osd/PeeringState.cc

```diff
@@ -6227,6 +6227,13 @@ boost::statechart::result PeeringState::Active::react(const AllReplicasActivated
     pl->send_pg_created(pgid);
   }
 
+  psdout(1) << __func__ << " AllReplicasActivated : Peering->Activating->Active over"
+           << " PG_STATE_ACTIVE=" << ps->state_test(PG_STATE_ACTIVE)
+           << " PG_STATE_WAIT=" << ps->state_test(PG_STATE_WAIT)
+           << " PG_STATE_PEERED=" << ps->state_test(PG_STATE_PEERED)
```

Comment:
psdout() will prefix the log line with the pg's operator<< which includes the state string.  These lines aren't needed.

Reply:
It is to pair with the log of PeeringState::start_peering_interval() to explicitly tell the print when the peering is complete.

PeeringState::start_peering_interval() tell us the pg peering(new interval) begin time
AllReplicasActivated :  tell us the pg peering end time

```
PeeringState::start_peering_interval()
{
  psdout(1) << __func__ << " up " << oldup << " -> " << up
        << ", acting " << oldacting << " -> " << acting
        << ", acting_primary " << old_acting_primary << " -> " 
        << new_acting_primary
        << ", up_primary " << old_up_primary << " -> " << new_up_primary
        << ", role " << oldrole << " -> " << role
        << ", features acting " << acting_features
        << " upacting " << upacting_features
        << dendl;
}
```

Reply:
Yes, but I think all you need is

```
  psdout(1) << __func__ << " AllReplicasActivated : Peering->Activating->Active over" << dendl;
```

The other information is already present in the pg prefix added implicitly by psdout.

Reply:
> Yes, but I think all you need is
> 
> ```
>   psdout(1) << __func__ << " AllReplicasActivated : Peering->Activating->Active over" << dendl;
> ```
> 
> The other information is already present in the pg prefix added implicitly by psdout.

psdout(1) << __func__ << " AllReplicasActivated Activating complete" << dendl;